Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_bottles.yml

```diff
@@ -77,7 +77,7 @@
 - type: entity
   parent: DrinkBottleGlassBaseFull
   id: DrinkAbsintheBottleFull
-  name: Jailbreaker Verte
```

Comment:
This sounds like it was intended to be a proper noun, and proper nouns should remain capitalized.